Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/DefaultObjectSrg.azsli

```diff
@@ -40,16 +40,43 @@ ShaderResourceGroup ObjectSrg : SRG_PerObject
     //! Reflection Probe (smallest probe volume that overlaps the object position)
     struct ReflectionProbeData
     {
-        float3 m_aabbPos;
-        float3 m_outerAabbMin;
-        float3 m_outerAabbMax;
-        float3 m_innerAabbMin;
-        float3 m_innerAabbMax;
+        row_major float3x4 m_modelToWorld;
+        row_major float3x4 m_modelToWorldInverse; // does not include extents
+        float3 m_outerObbHalfLengths;
+        float3 m_innerObbHalfLengths;
         float m_padding;
         bool m_useReflectionProbe;
         bool m_useParallaxCorrection;
     };
 
     ReflectionProbeData m_reflectionProbeData;
     TextureCube m_reflectionProbeCubeMap;
+
+    float4x4 GetReflectionProbeWorldMatrix()
+    {
+        float4x4 modelToWorld = float4x4(
```

Comment:
why not `float4x4 modelToWorld = float4x4(m_reflectionProbeData.m_modelToWorld[0], ...)`?

Reply:
I followed the existing pattern that is used in the Srgs for these functions.  We could make a separate task to refactor the initialization.